torchdynamo/variables/tensor.py

```diff
@@ -145,32 +148,26 @@ def context():
                             lambda: cls.run_proxy(proxy, args, kwargs, nnmodule)
                         )
                 except RuntimeError as e:
-                    raise TorchRuntimeError() from e
+                    if use_fake_tensors and isinstance(e, DataDependentOutputException):
+                        if (
+                            config.capture_scalar_outputs
+                            and proxy.node.target == "item"
+                        ):
+                            example_value = torch.zeros(
+                                size=(), dtype=args[0].dtype
+                            ).item()
+                        else:
+                            unimplemented("data dependent operator")
```

Comment:
nit - can you add that it's .item here? either hardcoded or just via `f" ... {proxy.node.target}"` - makes it easier to find and debug, also makes the torchdynamo.explain graph breaks easier to grok.